testing/schedule_test.py

```diff
@@ -0,0 +1,171 @@
+import unittest
```

Comment:
I suggest renaming this to `mdp_gen_schedule_test.py`

Reply:
Addressed! Thanks a lot!